lab-lee/app/component/gallery/gallery-li/gallery-li.js

```diff
@@ -0,0 +1,22 @@
+'use strtict';
+
+require('./_gallery-li.scss');
+
+module.exports = {
+  template: require('./gallery-li.html'),
+  controller: ['$log', 'galleryService', GalleryLIController],
+  controllerAs: 'galleryLICtrl',
+  bindings: {
+    gallery: '<',
+  },
+};
+
+function GalleryLIController($log, galleryService) {
+  $log.debug('init galleryLICtrl');
+
+  this.showEditGallery = false;
+
+  this.deleteGallery = function() {
```

Comment:
Great job getting deleteGallery to work with a gallery on the scope of the controller :)